namespaces/http.yml

```diff
@@ -0,0 +1,60 @@
+version: 2016.03.10.0
+
+defaults:
+  type: string
+  doc_values: true
+  index: not_analyzed
+
+namespace:
+  name: http
+  type: group
+  description: >
+    Attributes common to http requests.
+  fields:
+
+    - name: client_ip
```

Comment:
Could this be ip v6?
we usually call this ipaddr4 https://github.com/ViaQ/elasticsearch-templates/blob/master/namespaces/_default_.yml#L43
ip address v6 are natively supported only in ES 5+ https://www.elastic.co/blog/indexing-ipv6-addresses-in-elasticsearch